src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+    private static final double DEFAULT_RADIUS = 10.0;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        Figure[] randomFigures = new Figure[] {
+                new Circle(),
+                new Square(),
+                new IsoscelesTrapezoid(),
+                new RightTriangle(),
+                new Rectangle()
+        };
+
+        final int randomIndex = random.nextInt(randomFigures.length);
```

Comment:
create constant FIGURES_COUNT for that
```suggestion
        final int figureNumber = random.nextInt(FIGURES_COUNT);
```